uk.co.compucorp.civicrm.hrleaveandabsences/api/v3/AbsenceType.php

```diff
@@ -44,3 +44,47 @@ function civicrm_api3_absence_type_delete($params) {
 function civicrm_api3_absence_type_get($params) {
   return _civicrm_api3_basic_get(_civicrm_api3_get_BAO(__FUNCTION__), $params);
 }
+
+
+/**
+ * AbsenceType.calculateToilExpiryDate Specification
+ *
+ * @param array $spec
+ *
+ * @return void
```

Comment:
no need for @return void. If the method doesn't return anything, you can omit the @return tag